public/scripts/editor/js/fc/load-project-files.js

```diff
@@ -67,37 +68,107 @@ define(function(require) {
   }
 
   function load(project, options, callback) {
-    var root = project.root;
+    var root = project.root || ANONYMOUS_USER_FOLDER;
     var authenticated = options.authenticated;
     var config = JSON.parse(JSON.stringify(options));
+
     config.root = root;
     config.fs = Bramble.getFileSystem();
     config.shell = new config.fs.Shell();
```

Comment:
`var shell = config.shell = new config.fs.Shell();` Probably do the same thing for `fs` too so that you can just use `shell` and `fs` vs. `config.shell` and `config.fs` everywhere